### DISCUSSION TITLE
feat: User guide

### DISCUSSION THREAD

**Discussion by maliksenpai** [acikkaynak/deprem-yardim-frontend#832] [Ideas]
## Define your request
It seems like a small information box is needed so that users can navigate the page more easily and understand the application better. Therefore, this information box was designed and coded within the team for this.

## Define the alternatives you have evaluated
The best way to inform the user without disturbing can be the information box.

## Additional Context
Screenshot:

![image](https://user-images.githubusercontent.com/37189443/217599854-60fa825e-2edf-4eee-8f21-c0b061a32bf6.png)

**Comment by maliksenpai** [comment]
Moreover the development has been completed

**Comment by aerdemsenturk** [comment]
Also, the methodology description could be added to this information box.

https://github.com/acikkaynak/deprem-yardim-frontend/issues/465

**Comment by maliksenpai** [comment]
We can also add it soon, i already took those descriptons from "çabucak-ogren.md" file. @aerdemsenturk

**Comment by khnbzkrt** [comment]
I can assume in the case of approval  👍

**Comment by maliksenpai** [comment]
@khnbzkrt I have already done the coding, i am waiting for the approval

**Comment by khnbzkrt** [comment]
@maliksenpai Okey dude, well done ❤🙏

**Comment by maliksenpai** [comment]
I have created the pr, could be merged in case of approval. PR: https://github.com/acikkaynak/deprem-yardim-frontend/pull/514

**Comment by maliksenpai** [comment]
Can you have a look at here? @eraygundogmus code and design is ready, a pr could be created as soon as gets approved. If not wanted, close it.

**Comment by EmirBoyaci** [comment]
cc: @usirin

**Comment by handekaratas96** [comment]
@canerakin111 @eraygundogmus We have talked this with Caner, we can create difficulty of use with new additions, We have to reopen this task to be more de explanatory @Batuhandgenc @senapalaa can you work together i can support if needed

**Comment by maliksenpai** [comment]
@handekaratas96 I can update if we can discuss the text. Code is ready, if the text is ready i can update again.

**Comment by maliksenpai** [comment]
My discord username: MalikSenpai#8572

**Comment by maliksenpai** [comment]
Design part is already designed for the team. Besides, i do not know if refactor needed.

**Comment by handekaratas96** [comment]
Hello, i think the same too. By the way is it pop up on mobile?

 @maliksenpai if you want we can do it like this @senapalaa  when you get the text prepared you can look in the test environment and decide, what do you think? @eraygundogmus

**Comment by maliksenpai** [comment]
@handekaratas96 Yes mobile is similar. Button is clicked and information is given on popup. It covers more space in mobile for clarity. Okey, we can work in pairs and test.

**Comment by berkaydesign** [comment]
Design is done for this issue. You can check it out on Figma. @maliksenpai 

![image](https://user-images.githubusercontent.com/65855467/218264014-1ce8c076-c469-447e-aa55-7076679a3e28.png)

**Reply by nejatcanunlu** [reply]
> Design is done for this issue. You can check it out on Figma. @maliksenpai
> 
> ![image](https://user-images.githubusercontent.com/65855467/218264014-1ce8c076-c469-447e-aa55-7076679a3e28.png)

This seems to be stale, it doesn't include the latest features on the live map. Can we update the content accordingly, or could you point me to whomever you've received the content from?

**Comment by maliksenpai** [comment]
@berkaydesign good work, I will work on it, thank you.

**Comment by fdemir** [comment]
@maliksenpai do u have any update?

**Comment by maliksenpai** [comment]
@fdemir I worked a little but I'm busy right now. But i will finish as soon as I can, probably tonight will finish.

**Reply by nejatcanunlu** [reply]
@maliksenpai any updates on this?

**Reply by maliksenpai** [reply]
@nejatcanunlu There is no update, issue has closed. Third time closed without any comment. I do not work anymore, if you want, you can reach any authorized person.

**Reply by nejatcanunlu** [reply]
Thanks for the update, will do!

**Comment by nejatcanunlu** [comment]
Status update for all: The content of the user guide is currently being rewritten to match all the new features that are live on the map, we're waiting for the product team to come up with the latest version to update accordingly.

**Comment by handekaratas96** [comment]
@nejatcanunlu @fdemir @oguzergul with the deploy of the new features, we can add the user guide to the map, the texts in the link can be used here 
https://docs.google.com/document/d/1RJ1rP2VEUGP6jBM_S1E1FijcGfWWgCGZ10eMQK4usSE/edit
@senapalaa @merve başak
